thoth/storages/graph/models.py

```diff
@@ -50,9 +50,9 @@ class PythonPackageVersion(Base, BaseExtension):
     package_version = Column(String(256), nullable=False)
     # Nullable if we have unparseable entries or entries comming from package-extract where we
     # were unable to detect these entries.
-    os_name = Column(String(256), nullable=True)
-    os_version = Column(String(256), nullable=True)
-    python_version = Column(String(256), nullable=True)
+    os_name = Column(String(256))
+    os_version = Column(String(256))
+    python_version = Column(String(256))
```

Comment:
[A database migration](https://github.com/thoth-station/storages#generating-migrations-and-schema-adjustment-in-deployment) will be needed for this. But I would suggest to wait with this change to make a proper change in components, we can break stuff in deployments.